### PR TITLE
Skip user case for case explode

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?189820

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/phone/data_providers/case/load_testing.py [hunk 1]
```diff
@@ -44,3 +44,6 @@ def append_update_to_response(response, update, restore_state):
         current_count += 1
         if current_count < restore_state.loadtest_factor:
             update = transform_loadtest_update(original_update, current_count)
+        #only add user case on the first iteration
```
Comment (orangejenny) [inline corehq/ex-submodules/casexml/apps/phone/data_providers/case/load_testing.py]: not quite following the purpose of this change - @czue does this look good to you?

Comment (czue) [inline corehq/ex-submodules/casexml/apps/phone/data_providers/case/load_testing.py]: :+1: yep this looks right.

**Comment by czue** [comment]
not a huge deal, but probably would be better to use the `USERCASE_TYPE` constant for this

**Comment by orangejenny** [comment]
will merge so mwellcare can use this, since feedback doesn't sound blocking, @emord can respond when he's back

**Comment by orangejenny** [comment]
although, thinking about it, india might not get deployed until mid-next week anyway

**Pull request merged**